readme/BREAKING_CHANGES.markdown

```diff
@@ -1470,4 +1470,26 @@ This removal of the configuration has no replacement.
 
 ### Why was this change made?
 
-Liferay decided to not support this configuration.
\ No newline at end of file
+Liferay decided to not support this configuration.
+
+---------------------------------------
+
+## Removed extension points in SolrClientManager
+- **Date:** 2023-July-4
+- **JIRA Ticket:** [LPS-180691](https://liferay.atlassian.net/browse/LPS-180691)
+
+### What changed?
+
+These extension points in `SolrClientManager` for the `SolrClientFactory` with type `CLOUD` or `REPLICATED` and `HttpClientFactory` with type `BASIC` or `CERT` are being removed.
+
+### Who is affected?
+
+This affects anyone is overriding the `SolrClientFactory` and `HttpClientFactory` with the types liferay provided.
+
+### How should I update my code?
+
+Should not override the `SolrClientFactory` and `HttpClientFactory` with the types liferay provided, use customizer types instead.
```

Comment:
@slnn Let's say "Register `SolrClientFactory` and `HttpClientFactory` with custom types, and update the configurations."

Reply:
Done